File: 0217-contains-duplicate/0217-contains-duplicate.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        #comment to test autocommit
        freq = Counter(nums)
        for num, freq in freq.items():
            if freq > 1:
                return True
        return False
        #return True if len(nums) != len(set(nums)) else False

File: 0217-contains-duplicate/0217-contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
      sort(nums.begin(), nums.end()); // sort 
      int n = nums.size();
      for(int i = 1; i < n; i++){       // then linear search
          if(nums[i]==nums[i-1])
            return true;
      }
      return false;
    }
};